### PR TITLE
JBTM-3597: Upgrading quarkus dependency version

### PR DESCRIPTION
**Pull request by mayankkunwar**
https://issues.redhat.com/browse/JBTM-3597

JDK11 !JDK8

!CORE !TOMCAT !AS_TESTS RTS !JACOCO !XTS !QA_JTA !QA_JTS_JACORB !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !PERF LRA !DB_TESTS !mysql !db2 !postgres !oracle

### CODE REVIEW

File: rts/pom.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@ Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
   <url>http://www.jboss.org/jbosstm</url>
 
   <properties>
-    <version.quarkus>1.8.3.Final</version.quarkus>
+    <version.quarkus>2.8.3.Final</version.quarkus>
```
[the remark this answers is not in the document]
Comment (mayankkunwar) [reply rts/pom.xml:34]: Sure @tomjenkinson I will do that.

**Comment by mayankkunwar** [comment]
>do you know if the LRA JDK 11 build number 723 failure is new? I can't find a report of it in Jira so could be introduced in this upgrade?: https://issues.redhat.com/issues/?jql=project%20%3D%20JBTM%20AND%20text%20~%20compensateMultiLevelNestedActivity

I am not aware of this one @tomjenkinson

**Comment by mayankkunwar** [comment]
> > > do you know if the LRA JDK 11 build number 723 failure is new? I can't find a report of it in Jira so could be introduced in this upgrade?: https://issues.redhat.com/issues/?jql=project%20%3D%20JBTM%20AND%20text%20~%20compensateMultiLevelNestedActivity
> > 
> > 
> > I am not aware of this one @tomjenkinson
> 
> If it is a new issue, are you sure it's not introduced by this PR? If you determine that it's not introduced by this PR please can you raise a JBTM so it can be tracked?

Sure Tom I will raise new JBTM because the failure is not related to PR changes. As the failure is in `LRA Coordinator jar` module which do not use Quarkus dependencies.

**Comment by mmusgrov** [comment]
JBTM-3590 blocks JBTM-3597 so I think this PR has been raised to early.

**Comment by mayankkunwar** [comment]
> @mayankkunwar the LRA JDK 11 build #726 was hanging with this change

@tomjenkinson we saw the similar behavior here as well when tried to execute the build with JDK11 and the build was hanging in `LRA tests: MicroProfile TCK` module, it is waiting indefinitely (on object Monitor). I think this issue is cause by Quarkus latest version.

Logs can be found here http://narayanaci1.eng.hst.ams2.redhat.com/job/btny-pulls-narayana/PROFILE=LRA,jdk=jdk11.latest,label=linux/722/consoleFull

**Comment by mayankkunwar** [comment]
> @mayankkunwar FYI after resolving a merge issue, I used the commit you have here to provide the update in the JBTM-3590 branch too: [6d83853](https://github.com/jbosstm/narayana/commit/6d838530a0226cfd706309bb94ec6b2c6e47dc1b)

So, I now I think this PR is no longer required. But we can just look into the CI failures that we encountered when this PR changes ran on our CI.

**Pull request closed without merging**